device/iot-device-samples/send-batch-events/README.md

```diff
@@ -0,0 +1,60 @@
+# Send Event Sample
+
+Sample application that uses the Azure IoT Java SDK to send telemetry messages to the
+Azure IoT Hub cloud service or to an Azure IoT Edge device. The sample demonstrates how to connect
+and send messages using a protocol of your choices as a parameter.
+
+## Build the sample
+
+```
+$> cd {sample root}
+$> mvn install -DskipTests
+```
+
+## Run the sample
+
+Listed below is the command to launch the sample along with a description of its arguments
+
+```
+$> cd {sample root}/send-event
+$> java -jar target/send-event-{version}-with-deps.jar "{connection string}" {number of messages to send} {protocol}  {path to trusted root ca}
```

Comment:
Again, make sure you update this readme to reflect the different jar file that gets generated when building this sample